examples/apps/cra-demo/README.md

```diff
@@ -104,40 +106,45 @@ const getContainerId = () => {
 };
 ```
 
-### 3.b Instantiate a service client instance
+### 3.b Configure the service client
 
-`TinyliciousClient` instance needs to be instantiated before use and can take an optional configuration object to changes settings such as default port. Depending on your deploy target, you might create and use different client packages.
+In this demo we will only be dealing with a local configuration, but in a production environment you could swap this out with a production configuration without changing your application.
 
 ```js
 // add below getContainerId
-const tinyliciousClient = new TinyliciousClient();
+const localConfig = {
+    tenantId: "local",
+    tokenProvider: new InsecureTokenProvider("tenantId", { id: "userId" }),
+    orderer: "http://localhost:7070",
+    storage: "http://localhost:7070",
+};
 ```
 
 ## 4. <a style="position: relative; top: 20px" name="init"></a> Get Fluid Data
```

Comment:
Is the style needed?

Reply:
it makes the jump links work properly

Reply:
![image](https://user-images.githubusercontent.com/1434956/125840772-610fb966-e0fc-43b3-aed7-ae9ec8b73b94.png)